context/ethereumContext.tsx

```diff
@@ -442,9 +442,11 @@ export const EthereumProvider: React.FC<{}> = ({ children }) => {
   const _loadPrecompiled = () => {
     const precompiled: IReferenceItem[] = []
 
-    getActivePrecompiles(common).forEach((address: Address) => {
+    const addressIterator = getActivePrecompiles(common).keys()
+    let result = addressIterator.next()
+    while (!result.done) {
```

Comment:
They changed the type, it now returns a map, so the function doesn't exist anymore.

Reply:
This is ok, but I think I would reach for a `for ... in` or `for ... of` statement for this usually. So instead of the while loop, you could do:
```
const precompiles = getActivePrecompiles(common)
for (const result of Object.values(precompiles)) {
    const meta = PrecompiledMeta as IReferenceItemMetaList
    ....
```

Reply:
I tried that, but it was not accepting it, so I had to change it. Some extra compiler option to enable

Reply:
Was it this message?
> Type 'IterableIterator<PrecompileFunc>' is not an array type or a string type. Use compiler option '--downlevelIteration' to allow iterating of iterators.ts(2569)

Do you remember if you tried `getActivePrecompiles(common).values()` or `Object.values(getActivePrecompiles(common)`? Because the first one will give you that error while the second won't

Reply:
Yes exactly. And it was the first one but with the `keys()`. If the second does not generate that error, does that mean it copies evrything? In that case I prefer the current version.